pkg/utils/test_utils_test.go

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package utils
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pravega/zookeeper-operator/pkg/apis/zookeeper/v1beta1"
+	"github.com/pravega/zookeeper-operator/pkg/zk"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test(t *testing.T) {
+	RegisterFailHandler(Fail)
```

Comment:
No need to register here, Register is required for one file per package

Reply:
Done